src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -146,8 +147,8 @@
     <BinPlaceDir Include="$(LibrariesNativeArtifactsPath)" ItemName="NativeBinPlaceItem" />
     <NativeBinPlaceItem Include="$(PkgSystem_Net_MsQuic_Transport)\runtimes\win10-$(TargetArchitecture)\native\*" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and
-                        ('$(TargetArchitecture)' == 'arm64' or '$(TargetArchitecture)' == 'arm') and '$(DotNetBuildFromSource)' != 'true'">
+  -->
+  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'windows' and '$(DotNetBuildFromSource)' != 'true'">
```

Comment:
Should the condition for transport package above be the exact opposite of this condition?

Reply:
Ah, the block above is commented out, so it does not matter.

Reply:
I'm not sure sure I understand the comment. It used to be (sort of). Now the block above is commented out and we get arm64, x64 and x86 from msquic (On Windows). On Unix, msquic is delivered as system package.

Reply:
yah. maybe there is better way.  I did not want to remove it since we will likely get back to it during 8.0...
We will get msquic 2.1.1 soon. This tests the waters if we _could_ use their binaries for all platforms. 
(and pick up https://github.com/microsoft/msquic/pull/2962 and https://github.com/microsoft/msquic/pull/2964 linking fixes)